packages/compass-aggregations/src/components/pipeline-builder-workspace/pipeline-as-text-workspace/pipeline-preview.tsx

```diff
@@ -115,17 +125,24 @@ export const PipelinePreview: React.FunctionComponent<PipelinePreviewProps> = ({
         {shouldShowCount && <Body>{`Sample of ${docCount} ${docText}`}</Body>}
       </div>
       <PreviewResults isLoading={isLoading} previewDocs={previewDocs} />
+      <div className={outputStageStyles} data-testid="output-stage-preview">
+        {isOutStage && <OutputStageBanner stageOperator="$out" />}
+        {isMergeStage && <OutputStageBanner stageOperator="$merge" />}
```

Comment:
Tiny nit: looks kinda weird that it's two separate props, it can be either one or another, right? Maybe something like `outStage: '$out' | '$merge' | null` will be a better interface here?